components/project-card/project-card.jsx

```diff
@@ -212,4 +212,24 @@ export default React.createClass({
       </div>
     );
   }
-});
+}
+ProjectCard.propTypes = {
+  id: PropTypes.number.isRequired,
+  creators: PropTypes.array,
+  description: PropTypes.string.isRequired,
+  featured: PropTypes.bool,
+  getInvolved: PropTypes.string,
+  getInvolvedUrl: PropTypes.string,
+  interest: PropTypes.string,
+  issues: PropTypes.arrayOf(React.PropTypes.string),
```

Comment:
I did search for `React.PropTypes` and got no matching result back. Weird. :nauseated_face: I should've been more careful!

Thanks for catching it @Pomax